### PR TITLE
📝 Changes on member profile

### PR DESCRIPTION
**Pull request by WRadoslaw**
#4238

### CODE REVIEW

File: packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx [hunk 1]
```diff
@@ -20,25 +19,25 @@ export const CopyAddressButton: FC<CopyAddressButtonProps> = ({ address, classNa
...
-    <StyledText
-      as="button"
-      variant={size === 'big' ? 't300' : 't100'}
-      color="colorText"
-      className={className}
-      onClick={handleCopyAddress}
-    >
-      {shortenString(address, 6, 4)}
-      <Tooltip text="Copy account address" placement="top">
+    <StyledTooltip hideOnClick={false} text={copyButtonClicked ? 'Copied!' : 'Copy account address'} placement="top">
```
Comment (drillprop) [inline packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx]: According to designs, this should be `Copy wallet address' now

Comment (drillprop) [reply packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx]: Also, I would change the position of the tooltip if it's possible. Placement should be top left or left 
image

image

File: packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx [hunk 2]
```diff
@@ -20,25 +19,25 @@ export const CopyAddressButton: FC<CopyAddressButtonProps> = ({ address, classNa
     if (!address) {
       return
     }
-    copyToClipboard(address, 'Account address copied to clipboard')
+    copyToClipboard(address)
     setCopyButtonClicked(true)
     setTimeout(() => {
       setCopyButtonClicked(false)
     }, 3000)
```
Comment (drillprop) [inline packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx]: A small thing, but according to the design icon should change after 2 seconds. Also, it's a little buggy. Sometimes the text changes immediately, maybe we should move the timeout to useEffect?  Tooltips like acting weird in our app, if it's hard to fix, let's leave it as it is.

**Review by drillprop — CHANGES_REQUESTED** [review]
We need small adjustments regarding the tooltip, otherwise looks good. 

Also, you need to resolve conflicts.

File: packages/atlas/src/views/viewer/MemberView/MemberActivity.tsx [hunk 3]
```diff
@@ -112,7 +112,7 @@ export const MemberActivity: FC<MemberActivityProps> = ({
   return (
     <section>
       {!loading && items.length === 0 ? (
-        <EmptyFallback title="No activity" subtitle="Go out there and explore!" variant="small" />
+        <EmptyFallback title="No activity" subtitle="Go out there and explore!" variant="large" />
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/MemberView/MemberActivity.tsx]:
```suggestion
        <EmptyFallback title="No activity" subtitle="This member hasn’t done anything yet." variant="large" />
```

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx (hunk 2 above)
Comment (WRadoslaw) [reply packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx]: I'll just block onClick for the duration of the timeout

**Review by drillprop — APPROVED** [review]
LGTM!

**Review by attemka — APPROVED** [review]

**Pull request merged**